### ISSUE TITLE
SC2148 on foo.bash without shebang

### ISSUE THREAD

**Issue by bryceschober** [timonwong/vscode-shellcheck#43]
Contrary to the command-line behavior of `shellcheck`, I get an error on a simple include-only bash "module" named `foo.bash` that doesn't have a shebang, like:
```bash
function do_foo() {
  echo "hello world"
}
```

running shellcheck from the command line yields:
```shell
$ shellcheck foo.bash
$ shellcheck --version
ShellCheck - shell script analysis tool
version: 0.6.0
license: GNU General Public License, version 3
website: https://www.shellcheck.net
$
```

According to [the SC2148 wiki](https://github.com/koalaman/shellcheck/wiki/SC2148), it should be fine with a `.bash` extension. It does drop the error if I add the alternative `# shellcheck shell=ksh`, but it seems that shouldn't be necessary.

**Comment by timonwong** [comment]
Probably a bug, I'll take a look this weekend. Thanks for reporting

**Comment by timonwong** [comment]
It should be fixed in v0.7.0
